dynamic_rest/viewsets.py

```diff
@@ -337,4 +337,91 @@ def list_related(self, request, pk=None, field_name=None):
 
 
 class DynamicModelViewSet(WithDynamicViewSetMixin, viewsets.ModelViewSet):
-    pass
+
+    ENABLE_BULK_PARTIAL_CREATION = (
+        dynamic_settings.get('ENABLE_BULK_PARTIAL_CREATION', False))
```

Comment:
`False`/`True` typo has been fixed. But I'm not sure about `settings` problem since we already discussed this. I just followed the rest of the code, e.g. `PAGE = dynamic_settings.get('PAGE_QUERY_PARAM', 'page')`. Do you want me to replace `from django.conf import settings` on `from dynamic_rest.conf import settings`?